ceph.spec.in

```diff
@@ -613,7 +613,7 @@ python-cephfs instead.
 # common
 #################################################################################
 %prep
-%autosetup -p1
+%setup -q -n %{name}-%{version}-%{release}
```

Comment:
@nitinkamble we have a pull request addressing this also, see https://github.com/ceph/autobuild-ceph/pull/77